### ISSUE TITLE
"Assets not loaded correctly" error unless using Tor

### ISSUE THREAD

**Issue by zaphbbrox** [trezor/trezor-suite#12008]
Unless connection via Tor is activated, my assets won't show. "Assets not loaded correctly" will show.

**Info:**

-   Remembered Wallet: yes
-   Suite Version: app
-   OS: ManjaroLinux, 23.1.4
-   Firmware Version: Trezor Model One, 1.12.1

**How to reproduce**

1. Open Trezor Suite
2. Open Wallet
3. Wait for assets to be displayed

**Expected behavior**
Assets should show, after initial steps.

**Additional context**
After reading #8666, I checked if there's a difference, when activating connections via Tor.  
When Tor option is activated, my assets will show, otherwise not.

**Comment by zaphbbrox** [comment]
Tbh, I don't need a fix for this issue, since I'm more than fine using Tor for Trezor.  
It's more an information for other users and of course for the devs _(maybe they want to look into it)_.  
If you need more information let me know.

**Comment by MiroslavProchazka** [comment]
Hey @zaphbbrox , thanks for reporting this issue. Can you please install the latest version and try to reproduce it? Thanks

**Comment by MiroslavProchazka** [comment]
Closing due to the inactivity. If something new occurs, please comment on this thread. Thank you.

**Comment by zaphbbrox** [comment]
Hi, sorry for the inactivity.
Just tried with the latest AppImage version (24.5.2).
Still the same.

**Comment by MiroslavProchazka** [comment]
Hey :)

Thank you for reaching out. I will need to check with the devs, but the first thing that comes to my mind is:

- Do you use a VPN?

- Your IP address/internet provider may be blocked by Cloudflare, which is why you can connect via Tor.

I will get back to you once I sync with the dev team.

**Comment by MiroslavProchazka** [comment]
Can you try to connect via VPN to some server outside your country of origin? (without Tor). It will help us determinate whether the origin of the issue lies in your internet entry point/provider. 

Thanks!

**Comment by zaphbbrox** [comment]
Tried with two different VPN connections, still the same.
And it did work without Tor a  while ago (don't know which version of Suite).

**Comment by zaphbbrox** [comment]
Ok, sorry for the fuzz, but suddenly it's working again without Tor.

**Comment by MiroslavProchazka** [comment]
Hey, glad to hear that! :) 

Thanks for the info. Leaving it closed for now, please let me know in case new issue occurs.
